adrs/001-planned-remediation-date.md

```diff
@@ -0,0 +1,53 @@
+# 001 - Add new Planned Remediation Date field to Finding model
```

Comment:
What is the thought of other maintainers? Having such a document does only make sense, if we want to continue writing adrs. Otherwise its a singular thing. I personally wouldn't need it and I can't imagine forcing other contributers to write these documents.